scripts/ci-build.sh

```diff
@@ -6,22 +6,26 @@
 # User warning!
 
 # A casual user will need to read the script to understand what the first
-# parameter shall be to make it run:
+# parameter shall be to make it actually run:
```

Comment:
Not sure that I like this kind of comment, if the script is meant for casual users, this should be documented at the top of the script IMHO. Just my .2$.

Reply:
Maybe the wording is strange, but the intended meaning is rather to say that it's **not** intended for casual users.  A casual user that attempts to run it will be forced to read it first...

The point here is that the script is not that user friendly, but that's not the comment but the code in lines 13 to 31, and that was put there deliberately.  The reason is  1) It's a little bit  dangerous script to use as a casual user, so therefore it really should not be run without reading and understanding it first,  and 2) Supporting CI build is the priority task.  (It's all written in 13-31 so I won't repeat more).  The comment is there to explain the code.